src/databricks/model_training.py

```diff
@@ -338,55 +301,25 @@ def convert_feature_dataframe_to_dict(
 
 # DBTITLE 1,Inspect predictions for one batch
 # print out model predictions just to make sure everything worked
-if DEBUG:
-    results = keras_model.predict(val_gen[0][0])
-    print(np.hstack([results[c] for c in train_gen.targets]))
+results = keras_model.predict(val_gen[0][0])
+print(np.hstack([results[c] for c in train_gen.targets]))
 
 # COMMAND ----------
 
 # DBTITLE 1,Inspect predictions using logged model
 # evaluate the unregistered model we just logged and make sure everything runs
-if DEBUG: 
-    print(run_id)
-    pred_df = model.score_batch(test_data=test_data, run_id=run_id)
-    pred_df.display()
+print(run_id)
+# mlflow.pyfunc.get_model_dependencies(model_uri=sm.get_model_uri(run_id=run_id))
+# Load the model using its registered name and version/stage from the MLflow model registry
+model_loaded = mlflow.pyfunc.load_model(model_uri=sm.get_model_uri(run_id=run_id))
```

Comment:
yeah that is fair... I think the reason I put it there is because failed at inference time for like 1000 different reasons so far, and most of the time, it means I did something wrong in training and have to retrain the model. So I figured it would be good if it failed there just in case I don't run the eval script and I think "oh yay successful model run" and then don't realize I need to retrain. I can nest it in the DEBUG if you feel strongly.

Reply:
I don't feel strongly on this point so will defer to your judgement here

Reply:
okay I'm going to keep for now